src/components/index/TheiaIDEDownloads.js

```diff
@@ -89,12 +89,17 @@ const TheiaIDEDownloads = () => (
                 
                 <div className="header__buttons">
                     <a className="btn btn--cta" href="https://www.eclipse.org/downloads/download.php?file=/theia/ide/latest/windows/TheiaIDESetup.exe&r=1" rel="noopener">Windows</a>
-                    <a className="btn btn--cta" href="https://www.eclipse.org/downloads/download.php?file=/theia/ide/latest/macos/TheiaIDE.dmg&r=1" rel="noopener">MacOS (x86)˟</a>
-                    <a className="btn btn--cta" href="https://snapcraft.io/theia-ide" rel="noopener">Linux (Snap)</a>
                     <a className="btn btn--cta" href="https://www.eclipse.org/downloads/download.php?file=/theia/ide/latest/linux/TheiaIDE.AppImage&r=1" rel="noopener">Linux (AppImage)</a>
+                    <a className="btn btn--cta" href="https://snapcraft.io/theia-ide" rel="noopener">Linux (Snap)</a>
+                    <a className="btn btn--cta" href="https://www.eclipse.org/downloads/download.php?file=/theia/ide/latest/macos/TheiaIDE.dmg&r=1" rel="noopener">MacOS (x86)¹</a>
+                </div>
+                 <div className="header__buttons">
+                    <a className="btn btn--cta" href="insert link" rel="noopener">MacOS (ARM, <i>experimental</i>)²</a>
                  </div>
                  <center>
-                 <i>˟ The Mac x86 version might run slow and unstable on recent ARM systems. We are working on a dedciated build for ARM architectures on Mac.</i>
+                 <i>¹ The Mac x86 version might run slow and unstable on recent ARM systems. Please try our new dedicated ARM build.</i>
+                 <br></br>
+                 <i>² The Mac ARM version is new, experimental and currenlty still unsigned (see <a href="https://support.apple.com/guide/mac-help/open-a-mac-app-from-an-unknown-developer-mh40616/mac">here</a> how to start it). We are working on providing a signed version asap.</i>
```

Comment:
```suggestion
<i>² The Mac ARM version is new, experimental and currently still unsigned. Newer versions of MacOS set the `quarantine` attribute on programs downloaded from the internet and refuse to run the application. I order to still run the build, you will have to clear the `quarantine` attribute from the application. In a terminal, type 
```
xattr -d -r com.apple.quarantine /Applications/TheiaIDE.app
```
after you have installed Theia. The Theia app should now start normally. We are working on providing a signed version asap.</i>
```